src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -34,154 +37,250 @@
   private String userStaff = credentials.userNameStaff;
   private String userStaffPassword = credentials.passwordStaff;
 
-  private ChatPageObject openChatForUser(String userName, String password) {
+  private final String currentBrowserTab() {
+    return driver.getWindowHandle();
+  }
+
+  private ChatPage openChatForUser(String userName, String password) {
     WikiBasePageObject base = new WikiBasePageObject();
     base.loginAs(userName, password, wikiURL);
-    return base.openChat(wikiURL);
+    return new ChatPage().open();
   }
 
-  @DontRun(env = {"preview", "dev", "sandbox"})
-  @Test(groups = {"Chat_001", "Chat", "ChatTests"})
-  @RelatedIssue(issueID = "MAIN-6071")
-  public void Chat_001_twoUserEnterChat() {
-    ChatPageObject chatUserOne = openChatForUser(userOne, userOnePassword);
-    chatUserOne.verifyChatPage();
+  @DontRun(env = {"preview", "dev"})
```

Comment:
are we sure chat is not working on preview? I think it's fixed, it looks like it works -> http://preview.mediawiki119.wikia.com/wiki/Special:Chat